app/models/ndbn_member.rb

```diff
@@ -0,0 +1,16 @@
+# == Schema Information
+#
+# Table name: ndbn_members
+#
+#  account_name   :string           not null
+#  created_at     :datetime         not null
+#  updated_at     :datetime         not null
+#  ndbn_member_id :bigint           not null, primary key
+#
+class NDBNMember < ApplicationRecord
+  self.primary_key = "ndbn_member_id"
+
+  validates :ndbn_member_id, presence: true
```

Comment:
We could combine these `validates :ndbn_member_id, presence: true, uniqueness: true` but it is a small enough model and never likely to get bigger so readability is probably better.